tasks/coverage/typescript.snap

```diff
@@ -1454,53 +1434,17 @@ Expect to Parse: "classes/propertyMemberDeclarations/autoAccessor4.ts"
  6 │     accessor 1 = 1;
    ╰────
   help: Try insert a semicolon here
-Expect to Parse: "classes/propertyMemberDeclarations/autoAccessor5.ts"
+Expect to Parse: "classes/propertyMemberDeclarations/autoAccessorAllowedModifiers.ts"
 
   × Automatic Semicolon Insertion
-   ╭─[classes/propertyMemberDeclarations/autoAccessor5.ts:3:1]
- 3 │ class C1 {
- 4 │     accessor ["w"]: any;
-   ·             ┬
-   ·             ╰── Expected a semicolon or an implicit semicolon after a statement, but found none
- 5 │     accessor ["x"] = 1;
-   ╰────
+    ╭─[classes/propertyMemberDeclarations/autoAccessorAllowedModifiers.ts:14:1]
+ 14 │     accessor #j: any;
+ 15 │     accessor "k": any;
+    ·             ┬
```

Comment:
lol what's this grammar

Reply:
This seems like an easy fix, do you mind fixing this as well?

Reply:
> lol what's this grammar

Magic javascript 😂

> This seems like an easy fix, do you mind fixing this as well?

Ok !

Reply:
Checked double quote in [66e1047](https://github.com/Boshen/oxc/pull/63/commits/66e1047f22a8a8db6e488ed5176621665e77cb62)